addons/miscFixes/patchCUP/config.cpp

```diff
@@ -196,6 +196,59 @@ class CfgVehicles {
     class CUP_Tigr_M_KORD_Base: CUP_Tigr_Base {
         CUP_MAINTURRET_DISABLE_ATTENUATE;
     };
+    
+    // CUP UH-60M
+    class Helicopter; // might need to inherit from Air; per the CH-47 config?
+    class Helicopter_Base_F: Helicopter {};
+    class Helicopter_Base_H: Helicopter_Base_F {};
```

Comment:
We shouldn't need to include the entire inheritance here.
```suggestion
    class Helicopter_Base_H;
```